### PR TITLE
Update translation files

### PR DESCRIPTION
**Pull request by jorgesumle**

### CODE REVIEW

File: duden/locale/de_DE.po [hunk 1]
```diff
@@ -55,60 +55,75 @@ msgstr ""
...
 
-msgid "Found {} matching words. Use the -r/--result argument to specify which one to display."
-msgstr "{} passende Wörter genfunden. Benutze die Option -r/--result um das korrekte Ergebnis anzuzeigen."
+#: main.py:617
+msgid ""
+"Found {} matching words. Use the -r/--result argument to specify which one "
+"to display."
+msgstr ""
+"{} passende Wörter genfunden. Benutze die Option -r/--result um das korrekte "
+"Ergebnis anzuzeigen."
```
Comment (radomirbosak) [inline duden/locale/de_DE.po]: Is it necessary to wrap the lines?

File: duden/locale/de_DE.po [hunk 2]
```diff
@@ -55,60 +55,75 @@ msgstr ""
...
 
-#: main.py:606
+#: main.py:512
+msgid ""
+"Display n-th result in case of multple words matching the input. Starts at 1."
+msgstr ""
+
+#: main.py:515
+msgid "Enable fuzzy word matching."
+msgstr ""
```
Comment (radomirbosak) [inline duden/locale/de_DE.po]: Won't this cause that nothing gets displayed? I'd rather have an untranslated english string displayed than an empty string.

File: duden/locale/es_ES.po [hunk 3]
```diff
@@ -53,54 +53,74 @@ msgstr "No se pudo establecer la conexión. Comprueba tu conexión a Internet."
...
+
+#: main.py:617
+msgid ""
+"Found {} matching words. Use the -r/--result argument to specify which one "
+"to display."
+msgstr ""
+
+#: main.py:629
+msgid "No result with number {}."
+msgstr ""
```
Comment (radomirbosak) [inline duden/locale/es_ES.po]: I don't know enough spanish to translate these, but feel free to fill in the translations.

File: duden/locale/de_DE.po (hunk 1 above)
Comment (jorgesumle) [reply duden/locale/de_DE.po]: No, I don't generate the po files manually, by the way.

File: duden/locale/de_DE.po (hunk 2 above)
Comment (jorgesumle) [reply duden/locale/de_DE.po]: No, that will display the English string.

**Comment by radomirbosak** [comment]
Rebased and pushed to master. Thanks again for contributing :)

**Pull request closed without merging**